### ISSUE TITLE
The app content may disappear when rotating an emulator where a big document is open

### ISSUE THREAD

**Issue by mkulesh** [mkulesh/microMathematics#141]
Follow-up issue after #136:
After rotating an emulator where square_wave_oscillator example is open, the content disappears and app is not more usable.

Analysis: in the onCreate method, also check whether the input bundle is empty:
```
instanceStateEmpty = savedInstanceState == null || savedInstanceState.isEmpty();
```
instead of 
```
instanceStateEmpty = savedInstanceState == null;
```

**Comment by mkulesh** [comment]
Released in version 1.24.
